firmware/Core/Src/telecommands/flash_bank_telecommands_defs.c

```diff
@@ -0,0 +1,85 @@
+#include "telecommands/flash_bank_telecommands_defs.h"
+#include "flash_bank/flash_bank.h"
+#include "stm32l4xx_hal.h"
+#include "telecommands/telecommand_args_helpers.h"
+
+#include <stdio.h>
+#include <string.h>
+
+/// @brief Write data to the internal flash bank starting from address 0x08000000
+/// @param args_str
+/// - Arg 0: The data to write as a string
+/// @note This telecommand is only for testing purposes, it is purposfully not fully fleshed out
+/// as there is no intention on using this. Update as needed
+/// @note Currently, this is only writes all data given as a string.
+/// Thus, it is difficult to write integers directly
+uint8_t TCMDEXEC_flash_bank_write(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel, char *response_output_buf, uint16_t response_output_buf_len)
```

Comment:
Strongly recommend renaming this to `internal_flash_write_hex`. 

Strongly recommend using the argument as hex instead of as a c-string. Look at the other hex function to see how to do it.

Doesn't this function need an address to write at as an argument?